sites/partners/src/components/listings/ListingFormActions.tsx

```diff
@@ -195,60 +200,249 @@ const ListingFormActions = ({
+        elements.push(saveExitButton)
+
+        // admins can request changes on pending review listings
+        if (isListingApprover && listing.status === ListingStatus.pendingReview)
+          elements.push(requestChangesButton)
+
+        // all users can unpublish a closed listing
+        if (listing.status === ListingStatus.closed) {
+          elements.push(unpublishButton)
+        }
+
+        // all users can close or unpublish open listings
+        if (listing.status === ListingStatus.active) {
+          elements.push(closeButton)
+          elements.push(unpublishButton)
+        }
+
+        const lotteryResults = listing?.events?.find(
+          (event) => event.type === ListingEventType.lotteryResults
         )
-        elements.push(viewPostedResultsButton(eventUrl))
+
+        // all users can manage lottery results on closed listings
+        if (lotteryResults) {
+          elements.push(editPostedResultsButton(lotteryResults))
+        } else if (listing.status === ListingStatus.closed) {
+          elements.push(postResultsButton)
+        }
+
+        elements.push(cancelButton)
       }
+      return elements
     }
 
-    // new unsaved listing
-    if (type === "add") {
-      elements.push(publishButton)
-      elements.push(saveDraftButton)
-      elements.push(cancelButton)
-    }
+    const getCoreActions = () => {
+      const elements = []
+      // read-only form
+      if (type === "details") {
+        elements.push(editFromDetailButton)
+        elements.push(previewButton)
 
-    // listing saved at least once
-    if (type === "edit") {
-      elements.push(saveExitButton)
+        if (listing.events.find((event) => event.type === ListingEventType.lotteryResults)) {
+          const eventUrl = pdfUrlFromListingEvents(
+            listing?.events,
+            ListingEventType.lotteryResults,
+            process.env.cloudinaryCloudName
+          )
+          elements.push(viewPostedResultsButton(eventUrl))
+        }
+      }
 
-      if (listing.status === ListingStatus.pending || listing.status === ListingStatus.closed) {
+      // new unsaved listing
+      if (type === "add") {
+        elements.push(saveDraftButton)
         elements.push(publishButton)
+        elements.push(cancelButton)
       }
 
-      if (listing.status === ListingStatus.active) {
-        elements.push(closeButton)
-        elements.push(unpublishButton)
-      }
+      // listing saved at least once
+      if (type === "edit") {
+        if (listing.status === ListingStatus.pending) {
+          elements.push(publishButton)
+        }
+        if (listing.status === ListingStatus.closed) {
+          elements.push(reopenButton)
+        }
+        elements.push(saveExitButton)
+
+        if (listing.status === ListingStatus.active) {
+          elements.push(closeButton)
+        }
+
+        if (listing.status === ListingStatus.closed || listing.status === ListingStatus.active) {
```

Comment:
This is the only line changed for the core behavior, the rest is just from moving code around. The core change is that you can now see `Unpublish` on closed listings in addition to open listings.